### DISCUSSION TITLE
资源中心配置 - 配置对象存储 S3报错

### DISCUSSION THREAD

**Discussion by waynecookie** [apache/dolphinscheduler#13910] [Q&A]
[ERROR] 2023-04-11 16:43:15.002 +0800 org.springframework.boot.SpringApplication:[824] - Application run failed
org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'loginController': Unsatisfied dependency expressed through field 'authenticator'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'authenticator' defined in class path resource [org/apache/dolphinscheduler/api/security/SecurityConfig.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.apache.dolphinscheduler.api.security.Authenticator]: Factory method 'authenticator' threw exception; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'org.apache.dolphinscheduler.api.security.impl.pwd.PasswordAuthenticator': Unsatisfied dependency expressed through field 'userService'; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'usersServiceImpl': Unsatisfied dependency expressed through field 'storageOperate'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'storageOperate' defined in class path resource [org/apache/dolphinscheduler/service/storage/StoreConfiguration.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.apache.dolphinscheduler.service.storage.StorageOperate]: Factory method 'storageOperate' threw exception; nested exception is java.lang.NoClassDefFoundError: com/amazonaws/services/s3/model/AmazonS3Exception

**Comment by baofang111** [comment]
You should configure your resources in your Common.properties file with resources such as S3 

resource.storage.type ....
